.travis.yml

```diff
@@ -7,31 +7,38 @@ cache:
     - $HOME/.composer/cache/files
     - $HOME/symfony-bridge/.phpunit
 
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.1
-    - php: 7.2
+php:
+  - 7.1
+  - 7.2
 
 env:
   global:
-    - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
-    - SYMFONY_PHPUNIT_VERSION=5.7
+    - SYMFONY_PHPUNIT_VERSION=6.5
 
 before_install:
   - phpenv config-rm xdebug.ini
   - composer self-update
```

Comment:
Already execute by default, can be removed.
Execueted twice: https://travis-ci.org/PrestaShop/prestonbot/jobs/400949712#L476